srcpkgs/betterlockscreen/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'betterlockscreen'
+pkgname=betterlockscreen
+version=3.0.1
+revision=1
+archs=noarch
+depends="i3lock-color ImageMagick bc xrandr feh xdpyinfo"
+short_desc="Sweet looking lockscreen for linux system"
+maintainer="Aleksander Zien <Aleks@nderZien.com>"
+license="MIT"
+homepage="https://github.com/pavanjadhaw/betterlockscreen"
+distfiles="https://github.com/pavanjadhaw/betterlockscreen/archive/${version}.tar.gz"
+checksum=9b80af4b93e0b35bc916a584522ecf9eb39414c8010a2e4f2bb6941fdc5faf28
+
+do_install() {
+    vbin ${pkgname} ${pkgname}
```

Comment:
the second pkgname is not required, the binary is installed with the name given, the second argument is for when you want the binary to have another name.